tests/Mock/IgnWfsMockClient.php

```diff
@@ -26,42 +26,85 @@ private function handleRequests(string $method, string $url, array $options): Mo
 
     private function getWfsMock($options): MockResponse
     {
-        if (str_contains($options['query']['cql_filter'], "strStripAccents(strReplace(nom_minuscule, '-', ' ', true))=strStripAccents(strReplace('rue saint victor', '-', ' ', true))")) {
-            $body = [
-                'features' => [
-                    [
-                        'geometry' => [
-                            'type' => 'MultiLineString',
-                            'coordinates' => [
-                                [
-                                    [3.06773711, 50.65302845],
-                                    [3.06770772, 50.65320974],
-                                    [3.0676489, 50.65356155],
-                                    [3.06761116, 50.65383438],
-                                    [3.06756215, 50.65413324],
-                                    [3.06756371, 50.65424093],
+        if ($options['query']['TYPENAME'] === 'BDTOPO_V3:route_numerotee_ou_nommee') {
+            if (str_contains($options['query']['cql_filter'], "strStartsWith(numero, 'D32')=true AND gestionnaire='Ardennes' AND type_de_route='Départementale'")) {
```

Comment:
(Détail) Dans #669 je me suis dit qu'il était plus pérenne de juste checker la présence de valeurs clés, par ex

```suggestion
            if (str_contains($options['query']['cql_filter'], "'D32'")) {
```